src/char/char.cpp

```diff
@@ -1221,6 +1241,21 @@ int char_mmo_char_fromsql(uint32 char_id, struct mmo_charstatus* p, bool load_ev
 	StringBuf_AppendStr(&msg_buf, " hotkeys");
 #endif
 
+	if (SQL_ERROR == SqlStmt_Prepare(stmt, "SELECT `itemid` FROM `%s` WHERE `char_id`=?", schema_config.favs_db)
+		|| SQL_ERROR == SqlStmt_BindParam(stmt, 0, SQLDT_INT, &char_id, 0)
+		|| SQL_ERROR == SqlStmt_Execute(stmt)
+		|| SQL_ERROR == SqlStmt_BindColumn(stmt, 0, SQLDT_INT, &fav, 0, NULL, NULL)
+		) SqlStmt_ShowDebug(stmt);
+	noffav = 0;
+	while (SQL_SUCCESS == SqlStmt_NextRow(stmt))
+	{
+		if (noffav < MAX_FAVORITES)
+		{ p->favs[noffav] = fav; noffav++;  }
+		else
+		{ ShowWarning("mmo_char_fromsql: Attempt to load more favorites than MAX_FAVORITES!");	}
+	}
```

Comment:
Should look something like this
```suggestion
		if (noffav < MAX_FAVORITES) { 
			p->favs[noffav] = fav; noffav++;  
		} else {
			ShowWarning("mmo_char_fromsql: Attempt to load more favorites than MAX_FAVORITES!");	
		}
	}
```